Cargo.toml

```diff
@@ -66,9 +66,9 @@ serde = "1.0.0"
 serde_json = "1.0.0"
 
 # Examples
-tokio = { version = "0.2", features = ["dns", "macros", "rt-core", "sync", "tcp"] }
+tokio = { version = "0.3", features = ["rt-multi-thread", "macros", "sync", "net"] }
```

Comment:
Since these are the `dev-dependencies` it is probably to just enable the `full` feature, to get all the features for tests and examples.

Reply:
That would increase CI times.

Reply:
That is true. So it's a question of convenience vs. CI-time 😉